src/ontology/config/mondo-excluded-values.yml

```diff
@@ -0,0 +1,2402 @@
+exclusions:
```

Comment:
Good to see this file added back. It was removed when #93 was merged into `develop`. It can be found in the commit before that [here](https://github.com/monarch-initiative/mondo-ingest/blob/7d346079a929e72a8eecaa5117db7f5f72595b09/src/ontology/config/mondo-excluded-values.yml).

Reply:
If this file was re-named, what code needs to change to reflect this here?

Reply:
Since Nico didn't make any code changes, the code should be hooked up to `mondo-exclusion-configs.yml`.

I mis-wrote earlier. I said that `mondo-exclusion-configs.yml` was the old file. I should've said: It's what `mondo-exclusion-values.yml` was renamed to a few months ago.

The code is currently hooked up to `mondo-exclusion-configs.yml`:
https://github.com/monarch-initiative/mondo-ingest/blob/eb1bf1eaf27dfb11095dfe5400101d5677599454/src/ontology/mondo-ingest.Makefile#L622-L627

Reply:
Ok, I see [mondo-exclusion-configs.yml](https://github.com/monarch-initiative/mondo-ingest/blob/dev-new/src/ontology/config/mondo-exclusion-configs.yml) exists in `dev-new` so I agree, this file (`mondo-excluded-values.yml`) should be deleted.